README.md

```diff
@@ -15,6 +15,39 @@ for Windows' [Google Group](http://groups.google.com/group/git-for-windows),
 and [contribute bug
 fixes](https://github.com/git-for-windows/git/wiki/How-to-participate).
 
+To build Git for Windows, please either install [Git for Windows'
+SDK](https://gitforwindows.org/#download-sdk), start its `git-bash.exe`, `cd`
+to your Git worktree and run `make`, or open the Git worktree as a folder in
+Visual Studio.
+
+To verify that your build works, use one of the following methods:
+
+- If you want to test the built executables within Git for Windows' SDK,
+  prepend `<worktree>/bin-wrappers` to the `PATH`.
+- Alternatively, run `make install` in the Git worktree.
```

Comment:
Maybe add something like:

```
This will install the locally built Git executables in <where>....

Warning: this will replace your currently installed Git commands
with your locally built versions.  If there is a bug or problem with
your new version, you may break your development environment
and .... details...details...details.  So only use this method if you
have high confidence in your locally built version.
```
_I'm thinking about how the Makefile
runs git.exe to get the version and generate the build, so if git.exe is
broken, they may get stuck without a re-install..._

Reply:
We should add details for an alternative to install to a different directory.  Something of the form:

```
make prefix=/c/foo install
export PATH=/c/foo/bin:$PATH
```

and syntax for updating the CMD path to prepend "C:/foo/bin/".

Reply:
While I understand the sentiment to go into deeper detail, I would contend that the `README.md` is not the best place for a deep dive.

My intention here is not to explain how these things work internally, but to describe what people need to do to get on with their work...